### DISCUSSION TITLE
WEB 2024 - Feedback 1

### DISCUSSION THREAD

**Discussion by Nortalle** [web-classroom/web-classroom.github.io#47] [Announcements]
Hey @web-classroom/web-vgx-students !

This short questionnaire will enable us to find out how you have experienced the course so far.

All data is anonymous and will only be used to improve the course.

Thank you for your cooperation!

[WEB 2024 - Feedback 1](https://framaforms.org/web-2024-feedback-1-1698933856)

**Comment by samuelroland** [comment]
@Nortalle pour info les 3 2ème champs plus longs sont maintenant apparus dans le formulaire :) merci !

![image](https://github.com/web-classroom/web-classroom.github.io/assets/47849646/3e9710a2-0c51-40f4-adce-7be487b68ce4)
